pkg/oci/static/file.go

```diff
@@ -46,6 +46,9 @@ func NewFile(payload []byte, opts ...Option) (oci.File, error) {
 		return nil, err
 	}
 
+	// Add annotations from options
+	img = mutate.Annotations(img, o.Annotations).(v1.Image)
```

Comment:
Instead of calling mutate again, I think you can add the annotations in the addendum on line 42?

```go
	img, err := mutate.Append(base, mutate.Addendum{
		Layer: layer,
                Annotations: annotations,
	})
```

Reply:
Ah, yes! fixed

Reply:
> sorry for the delayed review! left a couple comments :)

No worries!